src/combine_variables.cpp

```diff
@@ -90,8 +94,16 @@ SEXP inds_combine(CharacterVector vars, ListOf<IntegerVector> xs) {
     xs_names = raw_names;
   }
 
+  // Find the first vector that is not all zero
+  int first = 0;
+  for (; first < xs.size(); first++) {
+    if (!all_zero(xs[first])) break;
```

Comment:
Could you place control flow statements on their own line please (with curly braces).

Reply:
Should I also do that for existing code ?

Reply:
Thanks. No need to change other code, I might remove the Rcpp dependency in the next release so I'll update style then.